bittytax/conv/parsers/celsius.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# (c) Nano Nano Ltd 2020
+
+import sys
+from decimal import Decimal
+
+from colorama import Fore, Back
+
+from ...config import config
+from ..out_record import TransactionOutRecord
+from ..dataparser import DataParser
+from ..exceptions import UnexpectedTypeError
+
+WALLET = "Celsius"
+
+
+def parse_celsius(data_row, parser, _filename):
+    in_row = data_row.in_row
+    data_row.timestamp = DataParser.parse_timestamp(in_row[1])
+
+    if in_row[8] != "Yes" and not config.args.unconfirmed:
+        sys.stderr.write("%srow[%s] %s\n" % (
+            Fore.YELLOW, parser.in_header_row_num + data_row.line_num, data_row))
+        sys.stderr.write("%sWARNING%s Skipping unconfirmed transaction, "
+                         "use the [-uc] option to include it\n" % (
+                             Back.YELLOW+Fore.BLACK, Back.RESET+Fore.YELLOW))
+        return
+
+    transaction_type = in_row[2]
+    symbol = in_row[3]
+    quantity = Decimal(in_row[4])
+    value = abs(Decimal(in_row[5])) if config.CCY == "USD" else None
```

Comment:
This seemed like the most reasonable approach, as if the config currency is `USD` we might as well use the precalculated values. Otherwise, the fiat values will be calculated automatically as normal.

Reply:
I think I'm going to add a new parser method which converts these USD values into the fiat currency specified by config.CCY.

There's a few parsers which could benefit from using the precalculated values.